src/util/timer.hpp

```diff
@@ -0,0 +1,36 @@
+#pragma once
+#include <chrono>
+
+namespace big
+{
+	class timer
+	{
+	public:
+		explicit timer(std::chrono::milliseconds delay) :
```

Comment:
The project allows for things like script yields to pass any duration type, strongly typing it to millisecond kind of pigeon-holes this class, but it's really up to @Yimura if it should be changed, as I wouldn't personally use this class if I needed something to keep track of time, it's not extensible.